WolvenKit.App/Controllers/RED4Controller.cs

```diff
@@ -364,47 +364,49 @@ private void CompileTweakFiles(Cp77Project cp77Proj)
                 _loggerService.Error(e);
             }
 
-            var tweakFiles = Directory.GetFiles(cp77Proj.TweakDirectory, "*.tweak", SearchOption.AllDirectories);
+            var tweakFiles = Directory.GetFiles(cp77Proj.TweakDirectory, "*.yaml", SearchOption.AllDirectories);
             foreach (var f in tweakFiles)
             {
-                var text = File.ReadAllText(f);
+//                var text = File.ReadAllText(f);
                 var folder = Path.GetDirectoryName(Path.GetRelativePath(cp77Proj.TweakDirectory, f));
                 var outDirectory = Path.Combine(cp77Proj.PackedTweakDirectory, folder);
                 if (!Directory.Exists(outDirectory))
                 {
                     Directory.CreateDirectory(outDirectory);
                 }
-                var filename = Path.GetFileNameWithoutExtension(f) + ".bin";
+                var filename = Path.GetFileName(f);
                 var outPath = Path.Combine(outDirectory, filename);
-
-                try
-                {
-                    if (!Serialization.Deserialize(text, out var dict))
-                    {
-                        continue;
-                    }
-                    var db = new TweakDB();
-                    //flats
-                    foreach (var (key, value) in dict.Flats)
-                    {
-                        db.Add(key, value);
-                    }
-                    //groups
-                    foreach (var (key, value) in dict.Groups)
-                    {
-                        db.Add(key, value);
-                    }
-
-                    using var ms = new MemoryStream();
-                    using var writer = new TweakDBWriter(ms);
-                    writer.WriteFile(db);
-                    File.WriteAllBytes(outPath, ms.ToArray());
-                }
-                catch (Exception e)
-                {
-                    _loggerService.Error(e);
-                    continue;
-                }
+                File.Copy(f, outPath, true);
```

Comment:
Are errors handled upstream somewhere so that the user knows this is where it fails if it does?